src/renderer/html_handlebars/hbs_renderer.rs

```diff
@@ -903,28 +903,42 @@ fn hide_lines(content: &str) -> String {
     }
 
     let mut result = String::with_capacity(content.len());
-    for line in content.lines() {
+    let mut lines = content.lines();
+    let last_line = lines.next_back();
+    let mut handle_line = |line: &str, newline: &str| {
```

Comment:
Instead of using a closure, can this be simplified a bit by using a peekable iterator? Perhaps something like this:

```rust
    let mut lines = content.lines().peekable();
    while let Some(line) = lines.next() {
        // Don't include newline on the last line.
        let newline = if lines.peek().is_none() { "" } else { "\n" };
```

Reply:
Thanks, that's a nice idea!